MAINTAINERS.md

```diff
@@ -0,0 +1,15 @@
+# MAINTAINERS
+
+## Current Maintainers
+
```

Comment:
nit: it seems the order is somewhat random which is perhaps the best order (?), but we might want to note that the order is not significant.

Reply:
Good catch. I will alphabetize them by first name like I did for the rest of them!

Reply:
hm. Actually, it really should be name in the first column and username is the second. I will update.

Reply:
Updated. lmk what you think @rylev and I'll update the rest as well.

Reply:
Looks great! 👍